Undefined io.ReadAll in version 3.1.0

Hi everyone,

While trying to install the library I had the following error message
```
go: found github.com/mailjet/mailjet-apiv3-go/v3 in github.com/mailjet/mailjet-apiv3-go/v3 v3.1.0
github.com/mailjet/mailjet-apiv3-go/v3/resources
github.com/mailjet/mailjet-apiv3-go/v3/fixtures
github.com/mailjet/mailjet-apiv3-go/v3
# github.com/mailjet/mailjet-apiv3-go/v3
../../go/pkg/mod/github.com/mailjet/mailjet-apiv3-go/v3@v3.1.0/core.go:254:13: undefined: io.ReadAll
```

By default version 3.1.0 is installed (as being the latest). No problems at all with version 3.0.0.

Other thing that might be useful, when clicking on the `build passing` badge from the master branch I only see a successful CI pipeline run for the 3.0.0 version. There is nothing about version 3.1.0. 

## Project
The project is a single main.go file with the following content:
``` go
package main

import (
    "fmt"
    "github.com/mailjet/mailjet-apiv3-go/v3"
)

func main() {
    fmt.Println("Init client")
    mailjet.NewMailjetClient("", "")
    fmt.Println("Done")
}
```
There is also a go.mod file generated as follows.

## Installation steps
```
go mod init mytest
go get -v ./...
```

Generated file:
```
module mytest

go 1.15

require github.com/mailjet/mailjet-apiv3-go/v3 v3.1.0
```

As previously said when pinning down to version 3.0.0, the code builds and behaves as intended.

Cheers

--- a/core.go
+++ b/core.go
@@ -251,7 +251,7 @@ func checkResponseError(resp *http.Response) error {
 		var mailjetErr RequestError
 		mailjetErr.StatusCode = resp.StatusCode
 
-		b, err := io.ReadAll(resp.Body)
+		b, err := ioutil.ReadAll(resp.Body)
 		if err != nil {
 			mailjetErr.ErrorMessage = "unable to read response body"
 			mailjetErr.ErrorInfo = err.Error()

--- a/core.go
+++ b/core.go
@@ -27,7 +27,7 @@ const (
 // User-Agent is formated as "UserAgentBase/UserAgentVersion;runtime.Version()".
 const (
 	UserAgentBase    = "mailjet-api-v3-go"
-	UserAgentVersion = "3.0.0"
+	UserAgentVersion = "3.0.1"
 )
 
 const (

